pipeline-braille-scripts/dtbook-to-pef/src/main/resources/xml/xproc/dtbook-to-pef.convert.xpl

```diff
@@ -11,109 +11,117 @@
-    <!-- <p:option name="main-document-language" required="true"/> -->
-    <p:option name="hyphenation" required="true"/>
-    <p:option name="line-spacing" required="true"/>
-    <!-- <p:option name="tab-width" required="true"/> -->
-    <p:option name="capital-letters" required="true"/>
-    <!-- <p:option name="accented-letters" required="true"/> -->
-    <!-- <p:option name="polite-forms" required="true"/> -->
-    <!-- <p:option name="downshift-ordinal-numbers" required="true"/> -->
-    <p:option name="include-captions" required="true"/>
-    <p:option name="include-images" required="true"/>
-    <!-- <p:option name="include-image-groups" required="true"/> -->
-    <p:option name="include-line-groups" required="true"/>
-    <!-- <p:option name="text-level-formatting" required="true"/> -->
-    <!-- <p:option name="include-note-references" required="true"/> -->
-    <p:option name="include-production-notes" required="true"/>
-    <p:option name="show-braille-page-numbers" required="true"/>
-    <p:option name="show-print-page-numbers" required="true"/>
-    <p:option name="force-braille-page-break" required="true"/>
-    <p:option name="toc-depth" required="true"/>
-    <!-- <p:option name="ignore-document-title" required="true"/> -->
-    <!-- <p:option name="include-symbols-list" required="true"/> -->
-    <!-- <p:option name="choice-of-colophon" required="true"/> -->
-    <!-- <p:option name="footnotes-placement" required="true"/> -->
-    <!-- <p:option name="colophon-metadata-placement" required="true"/> -->
-    <!-- <p:option name="rear-cover-placement" required="true"/> -->
-    <!-- <p:option name="number-of-pages" required="true"/> -->
-    <!-- <p:option name="maximum-number-of-pages" required="true"/> -->
-    <!-- <p:option name="minimum-number-of-pages" required="true"/> -->
-    <!-- <p:option name="sbsform-macros" required="true"/> -->
+    <p:input kind="parameter" port="parameters" sequence="true">
+        <p:inline>
+            <c:param-set/>
+        </p:inline>
+    </p:input>
 
     <!-- Empty temporary directory dedicated to this conversion -->
     <p:option name="temp-dir" required="true"/>
 
+    <p:import href="http://www.daisy.org/pipeline/modules/common-utils/library.xpl"/>
     <p:import href="http://www.daisy.org/pipeline/modules/braille/common-utils/library.xpl"/>
     <p:import href="http://www.daisy.org/pipeline/modules/braille/css-utils/library.xpl"/>
     <p:import href="http://www.daisy.org/pipeline/modules/braille/pef-utils/library.xpl"/>
     <p:import href="http://www.daisy.org/pipeline/modules/dtbook-utils/library.xpl"/>
     <p:import href="http://www.daisy.org/pipeline/modules/file-utils/library.xpl"/>
     <p:import href="http://www.daisy.org/pipeline/modules/fileset-utils/library.xpl"/>
-    <p:import href="generate-toc.xpl"/>
     
     <p:variable name="lang" select="(/*/@xml:lang,'und')[1]"/>
     
+    <!-- Ensure that there's exactly one c:param-set -->
```

Comment:
In retrospect I should've used `p:parameters` here. It's not a step I use regularly so I had forgotten about it.